os/linux/weston/plugin/compositor-iahwc.c

```diff
@@ -1347,6 +1347,7 @@ static int create_output_for_connector(struct iahwc_backend *b) {
   b->iahwc_get_display_info(b->iahwc_device, 0, configs[0], IAHWC_CONFIG_DPIY,
                             &output->base.mm_height);
 
+  //FIXME Memory leak for iahwc_mode
```

Comment:
@csravelar Could you please explain why this is a mem leak :)? I am unable to find the mem leak.

Reply:
Hey @harishkrupo. so I wanted a second opinion on this because I personally cannot see what the issue is either. However, cppcheck gives an error about the following line claiming there is a mem leak.